mods/default/crafting.lua

```diff
@@ -339,6 +339,27 @@ minetest.register_craft({
 })
 
 minetest.register_craft({
+	output = 'default:skeleton_key',
+	recipe = {
+		{'default:gold_ingot'},
+	}
+})
+
+minetest.register_craft({
+	type = 'cooking',
```

Comment:
Just a little nitpick, but the cooking recipes here should probably belong with the other ones below to keep consistency.